src/core/singleinstanceguard.cpp

```diff
@@ -23,6 +23,8 @@ bool SingleInstanceGuard::tryRun()
     // this will attach to the old segment, then exit, freeing the old segment.
     if (m_sharedMemory.attach()) {
         qInfo() << "another instance is running";
+        // so try to show it?
```

Comment:
Make the comment a sentence:

```suggestion
        // So try to show it?
```